src/scripts/main.js

```diff
@@ -1,3 +1,10 @@
 'use strict';
 
-// write code here
+const divElem = document.querySelector('.wall');
+const spiderElem = document.querySelector('.spider');
+
+const wigthdivElem = divElem.clientWidth;
+const heightdivElem = divElem.clientHeight;
+
+spiderElem.style.top = `${heightdivElem * 0.5 - 25}px`;
+spiderElem.style.left = `${wigthdivElem * 0.5 - 25}px`;
```

Comment:
what does `25` mean here? please use `spiderElem` to get the proper height and width